Pipfile

```diff
@@ -17,16 +17,17 @@ certifi = "*"
 chardet = "*"
 configparser = "*"
 dj-database-url = "*"
-Django = "*"
+django = "*"
 env-tools = "*"
 gunicorn = "*"
 idna = "*"
-psycopg2 = "*"
+"psycopg2" = "*"
 pytz = "*"
 requests = "*"
 tini = "*"
-urllib3 = "*"
+"urllib3" = "*"
 whitenoise = "*"
+requests-mock = "*"
```

Comment:
I see you added `requests-mock` here:

Instead of making it a regular addition to the `Pipfile` it should be listed under the `dev-packages`. To achieve this the package can be installed using `pipenv install requests-mock --dev`. Using this the dev packages will only be installed if the virtual environment is created with `pipenv install --dev`. 

The `requests-mock` also seems to not be in the `Pipfile.lock`? Could you add the current version to the PR as well?

Reply:
@gedankenstuecke i have updated the changes.